### DISCUSSION TITLE
How to use mongoDB Query Builder??

### DISCUSSION THREAD

**Discussion by piyushpathar** [jenssegers/laravel-mongodb#2716] [Ideas] [answered]
what is the new version of use Jenssegers\Mongodb\Eloquent\Builder;

**Comment by GromNaN** [comment, marked as answer]
Are you looking for [MongoDB\Laravel\Eloquent\Builder](https://github.com/mongodb/laravel-mongodb/blob/4.1/src/Eloquent/Builder.php)?

In `mongodb/laravel-mongodb` 4.0, we moved the namespace from `Jenssegers\Mongodb` to `MongoDB\Laravel`.

You can find other changes in the [Upgrading guide](https://www.mongodb.com/docs/drivers/php/laravel-mongodb/current/upgrade/).

**Reply by piyushpathar** [reply]
Done, Thanks
